python-package/MANIFEST.in

```diff
@@ -4,9 +4,13 @@ include *.rst *.txt
 recursive-include lightgbm *.py *.txt *.so
 recursive-include compile *.txt *.so
 recursive-include compile/Release *.dll
-recursive-include compile/compute *
+recursive-include compile/compute/ *.txt
```

Comment:
I changed this line from `*` to `*.txt` to exclude files like `.appveyor.yml` while still including important files like `CMakeLists.txt`  and the license